src/Options/docs.js

```diff
@@ -255,3 +256,8 @@
  * @property {String} triggerBeforeError Log level used by the Cloud Code Triggers `beforeSave`, `beforeDelete`, `beforeFind`, `beforeLogin` on error. Default is `error`.
  * @property {String} triggerBeforeSuccess Log level used by the Cloud Code Triggers `beforeSave`, `beforeDelete`, `beforeFind`, `beforeLogin` on success. Default is `info`.
  */
+
+/**
```

Comment:
See https://github.com/parse-community/parse-server/blob/alpha/CONTRIBUTING.md#parse-server-configuration how to add a new option to the Parse Server config.